src/aspire/source/image.py

```diff
@@ -937,31 +937,38 @@ def normalize_background(self, bg_radius=1.0, do_ramp=True):
             LambdaXform(normalize_bg, bg_radius=bg_radius, do_ramp=do_ramp)
         )
 
-    def im_backward(self, im, start, weights=None, symmetry_group=None):
+    def im_backward(self, idx, im=None, weights=None, symmetry_group=None):
```

Comment:
This is the change I mention in the PR comments.  Can roll it back if the prior interface is more clear to others.